operatorhub/Makefile

```diff
@@ -7,6 +7,9 @@ $(BIN)/operator-sdk: | $(BIN) ; $(info $(M) getting operator-sdk)
 	@./operatorhub/tools/install_operator-sdk.sh $(BIN)
 
 .PHONY: operator-bundle
-operator-bundle: | $(OPERATOR_SDK)
+operator-bundle: | $(OPERATOR_SDK) openshift/release-artifacts/bundle/manifests
 	@$(info BUNDLE_ARGS: $(BUNDLE_ARGS))
-	@OPERATOR_SDK=$(OPERATOR_SDK) operatorhub/tools/bundle.py ${BUNDLE_ARGS}
+	@OPERATOR_SDK=$(OPERATOR_SDK) ./tools/bundle.py ${BUNDLE_ARGS}
+
+openshift/release-artifacts/bundle/manifests:
+	mkdir -p openshift/release-artifacts/bundle/manifests
```

Comment:
is this target necessary. 

the bundle generation will create the `bundle/manifests` directory.
so i think here `mkdir -p openshift/release-artifacts/` should be sufficient.

In addtion, the bundle generation script should be the one creating the release-artifacts directory if it doesn't exist

Reply:
Ah right, I took the latest folder, but most likely we only need to create `openshift/release-artifacts`